resources/geocoding/hy/374.txt

```diff
@@ -0,0 +1,184 @@
+# Copyright (C) 2011 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Originally generated from:
+# http://en.wikipedia.org/wiki/+374 [440755811]
+# http://www.spyur.am/en/armenia (2011-07-28)
+# http://www.phonebookoftheworld.com/phonebookofarmenia.htm (2011-07-28)
+# http://www.itu.int/oth/T020200000A/en (2013-03-06)
+# http://phone.beeline.am/ru/main/codes.wbp (2013-03-06)
+# with some translation for a small number of entries, and to fix the
+# spelling to be consistent.
+
+37410|Երևան/Ջրվեժ
```

Comment:
Thank you for your answer.

So, "am" is country code of Armenia, but language code of Amharic. Language code of Armenian is "hy" (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).

Is it ok for you if I will merge these two folders into hy and remove am folder?